src/actions/oldInvitationsActions.js

```diff
@@ -110,10 +110,15 @@ export const fetchOldInviteNotificationsAction = (theWalletId?: string = '') =>
     // clean up local contacts
     const acceptedConnectionsIds = groupedNotifications.connectionAcceptedEvent.map(({ id: acceptedConnectionId }) =>
       acceptedConnectionId);
+    const disconnectedConnectionsIds =
+      groupedNotifications.connectionDisconnectedEvent.map(({ id: disconnectedConnectionId }) =>
```

Comment:
without renaming it will be shorter
```
groupedNotifications.connectionDisconnectedEvent.map(({ id }) => id);
```

Reply:
2f92b64 Thanks, some naming changes are needed